meta-cube/recipes-core/lxc/files/lxc-overlayclean

```diff
@@ -1,5 +0,0 @@
-#!/bin/sh -
```

Comment:
I see why we are changing the sed operations to manipulations on files in /var/lib/lxc .. for exactly the reason I was asking about in my previous comment. To make changes where r/w is possible. So the answer to my other question is likely "we've already taken into account the r/w /var/lib/lxc"